src/System.Text.Json/tests/JsonDocumentTests.cs

```diff
@@ -3740,6 +3698,47 @@ private static string GetCompactJson(TestCaseType testCaseType, string jsonStrin
 
             return s_compactJson[testCaseType] = existing;
         }
+
+        [Fact]
+        public static void WriteNumberTooLargeScientific()
+        {
+            // This value is a reference "potential interoperability problem" from
+            // https://tools.ietf.org/html/rfc7159#section-6
+            const string OneQuarticGoogol = "1e400";
+
+            // This just validates we write the literal number 1e400 even though it is too
+            // large to be represented by System.Double and would be converted to
+            // PositiveInfinity instead (or throw if using double.Parse on frameworks
+            // older than .NET Core 3.0).
+            var buffer = new ArrayBufferWriter<byte>(1024);
+            var expectedNonIndentedJson = $"[{OneQuarticGoogol}]";
+            using (JsonDocument doc = JsonDocument.Parse($"[ {OneQuarticGoogol} ]"))
+            {
+                var options = new JsonWriterOptions
+                {
+                    Indented = false,
```

Comment:
Since this is just specifying the default options it can be entirely gotten rid of.

Reply:
done